pages/index.tsx

```diff
@@ -1,31 +1,39 @@
 import type { NextPage } from 'next';
 import Head from 'next/head';
 import Image from 'next/image';
+import Link from 'next/link';
 
 const Home: NextPage = () => {
   return (
-    <div className='flex min-h-screen flex-col items-center justify-center py-2'>
+    <div className='min-h-screenpy-2'>
```

Comment:
There should be a space between `min-h-screen` and `py-2`. Or remove them if they're not necessary.

Reply:
ok